syntaxes/template.json

```diff
@@ -32,7 +32,7 @@
     },
 
     "propertyBinding": {
-      "begin": "(\\[.*\\])(=)(\"|')",
+      "begin": "(\\[[ a-zA-Z0-9\\._\\-@]*\\])(=)(\"|')",
```

Comment:
This would not recognize the banana syntax, would it?
Could you please add a test case for that?

Reply:
No, it won't. Do you mean to add a test case that it doesn't support the banana syntax?

Reply:
I think this is missing a `$` character. Not used super often (except for observables which you probably wouldn't want to property bind), but probably still worth including since it's easy to type and a valid character for identifiers.

Reply:
True: https://github.com/angular/angular/blob/master/packages/language-service/src/completions.ts#L75